manifests/init.pp

```diff
@@ -37,21 +37,25 @@
     }
     'RedHat': {
 
-      include nfs::idmap
-
-      $default_nfs_service = 'nfs'
+      $default_nfs_package = 'nfs-utils'
```

Comment:
The package name didn't (5,6,7 are all using 'nfs-utils') it's just been refactored up to the generic RHEL layer rather than the 5/6/7 case statement.

Although for some reason the nfs service isn't being controlled under RHEL7...  I was under then impression that the "nfs" service still needed to be started, as well as rpcbind (I could be wrong)